Revit_Core_Engine/Query/SpatialBoundCondition.cs

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using Autodesk.Revit.DB.Structure;
+using BH.oM.Adapters.Revit.Enums;
+using BH.oM.Base.Attributes;
+using System;
+using System.ComponentModel;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Check if a room/space/area is bounded, unplaced, redundant, or not enclosed.")]
+        public static BoundCondition SpatialBoundCondition(this SpatialElement elem)
+        {
+            double area = Math.Round(elem.Area, 2);
```

Comment:
Why are you rounding to the 2nd digit? Is that an empirical choice? If not, then I would recommend passing `RevitSettings` to this method and use its `DistanceTolerance` property here, to keep consistent with the rest of the toolkit